src/AspNetCoreModuleV2/CommonLib/hostfxr_utility.cpp

```diff
@@ -142,15 +142,26 @@ HOSTFXR_UTILITY::GetStandaloneHostfxrParameters(
     return hr;
 }
 
+BOOL
+HOSTFXR_UTILITY::IsDotnetExecutable(STRU *struExecutablePath)
+{
+    if (struExecutablePath == NULL)
+    {
+        return FALSE;
+    }
+    return struExecutablePath->EndsWith(L"dotnet.exe") || struExecutablePath->EndsWith(L"dotnet");
```

Comment:
how about the scenarion of mydotnet.exe or mydotnet? We can fix it in next PR

Reply:
Yeah next PR.